sycl/plugins/esimd_cpu/pi_esimd_cpu.cpp

```diff
@@ -637,6 +375,32 @@ template <class KernelType, class KernelArgType, int DIMS> class libCMBatch {
   }
 };
 
+void sycl_get_cm_buffer_params(void *PtrInput, char **BaseAddr, uint32_t *Width,
+                               std::mutex *MtxLock) {
+  _pi_buffer *buf = static_cast<_pi_buffer *>(PtrInput);
+
+  *BaseAddr = cm_support::get_surface_base_addr(buf->SurfaceIndex);
+  *Width = static_cast<uint32_t>(buf->Size);
+
+  // '=' assignment causes build failure for std::mutex
+  std::memcpy(MtxLock, &(buf->mutexLock), sizeof(std::mutex));
+}
+
+void sycl_get_cm_image_params(void *PtrInput, char **BaseAddr, uint32_t *Width,
```

Comment:
```suggestion
static void sycl_get_cm_image_params(void *PtrInput, char **BaseAddr, uint32_t *Width,
```

Reply:
Will apply suggested fix - for `sycl_get_cm_buffer_params` also.

Reply:
Suggestion applied.